src/module/actor/entity.js

```diff
@@ -337,7 +337,7 @@ export class OseActor extends Actor {
     const label = game.i18n.localize(`OSE.roll.hd`);
     const rollParts = [actorData.hp.hd];
     if (actorType == "character") {
-      rollParts.push(actorData.scores.con.mod);
+      rollParts.push(actorData.scores.con.mod*actorData.details.level);
```

Comment:
Do we care about how this relates to OSE AF's multiclassing rules?

Reply:
I will add this to #270, which I just wrote up. For now I don't foresee being able to do this without some kind of enhancement to the leveling experience (and we're treating this as a bugfix)